iOS/SalesforceMobileSDK-UITest.xcodeproj/xcshareddata/xcschemes/TestLogin.xcscheme

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
```

Comment:
`xcuserdata` folder should be added to `.gitignore` - the binary above should NOT be included, i.e. `bpage.xcuserdatad`.